mypy_whitelist.txt

```diff
@@ -0,0 +1,17 @@
+pandas/core/dtypes/base.py
```

Comment:
These are all of the files that currently have some sort of hints in them. The motivation for this comes form this part in the documentation:

https://mypy.readthedocs.io/en/latest/running_mypy.html#reading-a-list-of-files-from-a-file

So thinking for initial CI runs we can whitelist the modules we want to run against, though this is ideally just a temporary file

Reply:
To be clear when running this from the project root you would do `mypy @mypy_whitelist.txt`

Reply:
@Naddiseo any chance you have experience with this? Reading from docs I think suggested approach will be to whitelist particular modules at the outset and slowly open up as more are added. 

I'd like to avoid having two files to control configuration here but I don't see an easy way in the `.ini` file to control which modules actually get checked

Reply:
@WillAyd, not with the whitelist approach. I went with the blacklist approach instead, and had a bunch of modules and packages with `ignore_errors` set.

Reply:
Thanks for the feedback. Was that done per package / module in the config file?

Reply:
Currently I'm just ignoring packages. But, I think at one point I was doing both, and I seem to remember doing a mixture where I'd ignore everything in a package except a specific file.
I think it looked like:

```ini
[mypy-package.subpackage]
ignore_errors=True
[mypy-package.subpackage.module]
ignore_errors=False
```
However, I don't remember if it worked or not.